java/packagescanner_test.go

```diff
@@ -10,6 +15,35 @@ import (
 	"github.com/quay/claircore/test"
 )
 
+// TestDisableAPI runs the java scanner over some layers known to have java
+// packages installed.
+func TestDisableAPI(t *testing.T) {
```

Comment:
what information did we find with this test?

Reply:
Sorry, can you rephrase the question?

(I elaborated a bit here but I'm not sure it answers your question: https://github.com/quay/claircore/pull/1158#issuecomment-1920237990)